build.gradle

```diff
@@ -9,6 +9,11 @@ buildscript {
 
 subprojects { Project subproject ->
     group "io.micronaut.micrometer"
+    repositories {
+        maven {
+            url 'https://repo.spring.io/milestone'
+        }
+    }
```

Comment:
this is needed to get milestone versions, but should be removed after 1.7.0 comes out.